### DISCUSSION TITLE
Why can't I find xrdp 0.9.0 version?

### DISCUSSION THREAD

**Discussion by Mr-LZG** [neutrinolabs/xrdp#3254] [Q&A] [answered]
Why can't I find xrdp 0.9.0? Was it deleted? Why was it deleted?

**Comment by matt335672** [comment]
I've done a bit of archeology here.

- There's references to 0.9.0 on the Internet
- Commit 6aed11a8cebd155e639e698c0aab5fc34b2f80dd (Update version to 0.9.1) shows a previous version of v0.9.0
- Commit 3baa5ffa231b2cec6cc1236ada5dd8b419309d70 (update version) shows the version being incremented from 0.7.0 to 0.9.0

In between these two commits I can't find any reference to a release in the git history:-

```
log 3baa5ffa231b2cec6cc1236ada5dd8b419309d70..6aed11a8cebd155e639e698c0aab5fc34b2f80dd | grep -i release
```

I suspect what has happened is one or more distros have used a devel tarball from in between these two commits with the version set to 0.9.0.

If you've got a deb or an rpm file we can try to work out exactly what's in it, if it's important. For example, you might be conducting supply chain integrity checks (which IMO is important).

**Reply by metalefty** [reply, marked as answer]
Short answer: xrdp 0.9.0 is not deleted, it just has not been released. 

Hi, v0.9.1 is the first release since I joined xrdp dev team.

> I suspect what has happened is one or more distros have used a devel tarball from in between these two commits with the version set to 0.9.0.

I remember that's correct! We didn't make releases periodically before v0.9.1. Some distro used a git snapshot for their package and the version is set to 0.9.0. "0.9.0" doesn't refer to a specific commit. So more than one source codes are called 0.9.0 unofficially. We have decided to use v0.9.1 for the first release of v0.9.x to avoid confusion.
